python/services/subsetservice/dmod/subsetservice/__main__.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import flask
+import json
+from dmod.modeldata import SubsetDefinition, SubsetHandlerImpl
+from . import name as package_name
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+subset_handler: SubsetHandlerImpl = None
+
+
+@app.route('/', methods=['GET'])
+def home():
+    return "<h1>DMoD Subset API</h1><p>This site is a prototype API for generating and retrieving model subsets.</p>"
+
+
+# A route to verify a catchment id is valid (i.e., exists in the known hydrofabric
+@app.route('/subset/cat_id_valid', methods=['POST'])
```

Comment:
One thought, might want to include the model in the url endpoint, i.e. `/subset/ngen/cat_id_valid`